src/mame/pacman/pacman.cpp

```diff
@@ -8263,6 +8289,7 @@ GAME( 1981, popeyeman,puckman,  pacman,   pacman,   pacman_state,  empty_init,
 GAME( 1980, pacuman,  puckman,  pacman,   pacuman,  pacman_state,  empty_init,    ROT90,  "bootleg (Recreativos Franco S.A.)", "Pacu-Man (Spanish bootleg of Puck Man)",                   MACHINE_SUPPORTS_SAVE ) // common bootleg in Spain, code is shifted a bit compared to the Puck Man sets. Title & Manufacturer info from cabinet/PCB, not displayed ingame
 GAME( 1980, crockman, puckman,  pacman,   pacman,   pacman_state,  empty_init,    ROT90,  "bootleg (Rene Pierre)",             "Crock-Man",                                                MACHINE_SUPPORTS_SAVE )
 GAME( 1980, crockmnf, puckman,  pacman,   pacman,   pacman_state,  empty_init,    ROT90,  "bootleg (Marti Colls)",             "Crock-Man (Marti Colls bootleg of Rene Pierre Crock-Man)", MACHINE_SUPPORTS_SAVE )
+GAME( 1980, puckmana, puckman,  pacman,   pacman,   pacman_state,  empty_init,    ROT90,  "bootleg (Alca)",                    "Puck Man (bootleg, Alca)",                                 MACHINE_SUPPORTS_SAVE )
```

Comment:
@ajrhacker has cleaned up a lot of the descriptions of bootlegs so they follow a common format.  Please change the description to “Puck Man (Alca bootleg)” so it follows the same format.

Reply:
Done! Thx!

Changed also "pacmanvg" to follow the common format (is the one I used as reference).